### PR TITLE
Add a Cloud Engineering Summit landing page

### PR DESCRIPTION
**Pull request by zchase**
This PR adds a cloud engineering summit landing page. The purpose of this page is to allow people to sign up for a notification for when registration opens and also provides a link to submit a CFP.

### CODE REVIEW

**Review by susanev — APPROVED** [review]
this looks fine, but if theres anyway to reduce the giant blocks of content down a bit it will make this easier much easier for folks to consume

File: themes/default/components/src/components/date-countdown/date-countdown.tsx [hunk 1]
```diff
@@ -47,12 +52,18 @@ export class DateCountdown {
...
 
     private renderCountdown() {
-        const { days, hours, minutes, seconds } = this.countdownData;
+        const { days, hours, minutes, seconds, distance } = this.countdownData;
         return <p class={this.textClass}>
-            <span>{ days }{this.renderValueLabel("days")} </span>
-            <span>{ hours }{this.renderValueLabel("hours")} </span>
-            <span>{ minutes }{this.renderValueLabel("minutes")} </span>
-            <span>{ seconds }{this.renderValueLabel("seconds")}</span>
+            { distance > 0 ?
```
Comment (cnunciato) [inline themes/default/components/src/components/date-countdown/date-countdown.tsx:57]: YES. 😂 Completely forgot to come back to this one. 

As a pure nit, I might suggest changing the name to like `remainingTimeInMS` or something more representative of what the value is, as `distance` sounds less like time and more like space. 😄

File: themes/default/content/cloud-engineering-summit/_index.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    - name: Build
+      topics:
+        - Reusable infrastructure practices
+        - Docker, containers, or Kubernetes
+        - Building React Native, cloud-based apps
+        - Serverless apps development
+        - Data science and ML apps
+        - Sharing your open source project
+      description: |
+        Modern cloud applications and architectures blur the lines between app infrastructure and cloud engineering teams adopting new tools and practices to tame complexity and accelerate delivery. They embrace that success in the modern cloud era requires some level of proficiency in both cloud infrastructure and applications. Whether they're full-stack or specialize in one area, all cloud engineers apply a software engineering mindset and practices to building and testing applications and the underlying cloud infrastructure. This includes using standard programming languages, applying software principles such as reusability and abstractions and testing, and leveraging the rich ecosystem of software development tools.
```
Comment (cnunciato) [inline themes/default/content/cloud-engineering-summit/_index.md]: Wording here seems like something may be missing -- is this maybe right?

```suggestion
        Modern cloud applications and architectures blur the lines between app infrastructure and cloud engineering teams adopting new tools and practices to tame complexity and accelerate delivery. Embracing that success in the modern cloud era requires some level of proficiency in both cloud infrastructure and applications. Whether they're full-stack or specialize in one area, all cloud engineers apply a software engineering mindset and practices to building and testing applications and the underlying cloud infrastructure. This includes using standard programming languages, applying software principles such as reusability and abstractions and testing, and leveraging the rich ecosystem of software development tools.
```

File: themes/default/layouts/page/cloud-engineering-summit.html [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+                    <div class="dot-background no-animate"></div>
+                </div>
+            </div>
+        </div>
+
+        <div class="container mx-auto">
+            <h2 class="text-center">Summit Tracks</h2>
+            <div class="text-center max-w-4xl mx-auto">
+                <p class="mb-12">
+                    We're inviting community members, including infrastructure (DevOps, SREs, infra engineers),
```
Comment (cnunciato) [inline themes/default/layouts/page/cloud-engineering-summit.html:54]: Without the parenthetical, this reads as `We're inviting community members, including infrastructure, platform engineers...`, so suggest putting `engineers` outside the parens:

```suggestion
                    We're inviting community members, including infrastructure (DevOps, SRE, infra) engineers,
```

File: themes/default/layouts/page/cloud-engineering-summit.html [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+                    platform engineers, application developers, full-stack engineers, and security professionals
+                    who are building modern cloud-based applications and infrastructure, to submit a talk. You
```
Comment (cnunciato) [inline themes/default/layouts/page/cloud-engineering-summit.html]: Again, super small nit, but it takes a _long_ time to get to `to submit a talk`, and it's an important part of the sentence. Might read better to bring it closer to the beginning, like "We're inviting everyone from the community to submit a talk, including ..." and then the list.

**Review by cnunciato — APPROVED** [review]
LGTM, couple small things to fix, and the rest just suggestions (and I agree these text blocks are pretty thick). 

We should probably add the usual padding around these sections, too, on mobile (the others all look fine):

image

File: themes/default/components/src/components/date-countdown/date-countdown.tsx (hunk 1 above)
Comment (zchase) [reply themes/default/components/src/components/date-countdown/date-countdown.tsx:57]: 😄

File: themes/default/layouts/page/cloud-engineering-summit.html (hunk 4 above)
Comment (zchase) [reply themes/default/layouts/page/cloud-engineering-summit.html]: 👍

File: themes/default/layouts/page/cloud-engineering-summit.html (hunk 3 above)
Comment (zchase) [reply themes/default/layouts/page/cloud-engineering-summit.html]: 👍

File: themes/default/content/cloud-engineering-summit/_index.md (hunk 2 above)
Comment (zchase) [reply themes/default/content/cloud-engineering-summit/_index.md]: 👍

**Pull request merged**